src/processing/IUser.java

```diff
@@ -0,0 +1,27 @@
+package processing;
+
+import java.util.*;
+import processing.utility.*;
+
+/**
+*
+* @author Himanshu Jain
+*/
+
+public interface IUser {
+    // this function is used to get the user details during initial setup from user through UI 
+    // if the board Id is null then we get a new board id from server and pass to UI
+    // parameter: userName - user name
```

Comment:
Please refer to the Javadoc formatting guidelines. Kindly make use of block tags
[Java Formatting](https://google.github.io/styleguide/javaguide.html#s7-javadoc)